packages/apidom-ls/src/config/codes.ts

```diff
@@ -695,8 +695,14 @@ enum ApilintCodes {
 
   OPENAPI3_1_OPENAPI_VALUE_PATTERN_3_1_0 = 7000100,
 
-  OPENAPI3_1_INFO = 7010000,
-  OPENAPI3_1_INFO_FIELD_SUMMARY_TYPE = 7010100,
+  OPENAPI3_1_OPEN_API = 7010000,
```

Comment:
Just a notice here: don't do this in the future - changing the code error numbers. This will shuffle the error numbers that people might be relaying on.

Reply:
noted. Should we create possibly unused codes anyways for OPENAPI3_1, which might help future devs not overlap/overwrite?

```
  OPENAPI3_1_OPEN_API_FIELD_WEBHOOKS_TYPE = 7010100, ---> 7010900 (or 7010800 if below change also made)
  OPENAPI3_1_OPEN_API_FIELD_JSON_SCHEMA_DIALECT_TYPE = 7010200, ---> 7011000 (or 7010900 if below change also made)
```

Is the following intended, where  `OPENAPI3_0_INFO_FIELD_VERSION_TYPE` is given the same code grouping as `OPENAPI3_0_OPEN_API_FIELD` instead of with other `OPENAPI3_0_INFO_FIELD`(s)?

```
OPENAPI3_0_OPEN_API_FIELD_EXTERNAL_DOCS_TYPE = 5010700,
OPENAPI3_0_INFO_FIELD_VERSION_TYPE = 5010800,
OPENAPI3_0_INFO_FIELD_VERSION_REQUIRED,

OPENAPI3_0_INFO = 5020000,
```

vs should be? 

```
OPENAPI3_0_OPEN_API_FIELD_EXTERNAL_DOCS_TYPE = 5010700,

OPENAPI3_0_INFO = 5020000,
OPENAPI3_0_INFO_FIELD_VERSION_TYPE = 5020600,
OPENAPI3_0_INFO_FIELD_VERSION_REQUIRED,

```

If you agree that the codes should be changed, I can create a new ticket for it. And if needed, make a breaking change note in an upcoming release.